### ISSUE TITLE
Various lavaland (and probably space) ruins have tiles exposed to space roundstart after #32860

### ISSUE THREAD

**Issue by deathride58** [tgstation/tgstation#32990]
Title. Thought this was a downstream issue for a little while until i actually checked the files in question and saw no diffs.

https://i.imgur.com/WYyeyFl.png screenshot from some dude on Citadel's discord, the file for the ruin downstream matches upstream. cba to hunt down all of the mapping issues introduced by #32860 

The ruins are fucked. Fix em

**Comment by WJohn** [comment]
Dupe of #27719

This has been around for a while.

**Comment by deathride58** [comment]
This is not a dupe of #27719

This is an issue caused by #32860, a look at the files for the ruins themselves show that they have /turf/open/space/basic tiles in places they shouldn't.

**Comment by ShizCalev** [comment]
Not a dupe.

**Comment by WJohn** [comment]
Oh! I did not see the space tiles under the tables there. I noticed the lava tile which is another issue.

**Comment by WJohn** [comment]
This apparently has cropped back up again? I've been getting players talking about this, but I haven't seen it myself.

**Comment by WJohn** [comment]
Oh, mistook this issue for another one, nevermind this one.
